src/components/nps/components/nps-scale.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react'
+import { range } from 'lodash-es'
+import clsx from 'clsx'
+
+const MIN = 0
+const MAX = 10
+
+export function NPSScale({ onSubmit }: { onSubmit?: (value: number) => void }) {
+  const [value, setValue] = useState<number | null>(null)
```

Comment:
I think we are using `undefined` for state empty state in most of the components. Can we use that?